crm_request/security/ir.model.access.csv

```diff
@@ -0,0 +1 @@
+admin_access_holiday_closure,admin_access_holiday_closure,model_holiday_closure,sales_team.group_sale_manager,1,1,1,1
```

Comment:
Your security file misses the header line which is defining the columns. Please add this at first:
```suggestion
id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
admin_access_holiday_closure,admin_access_holiday_closure,model_holiday_closure,sales_team.group_sale_manager,1,1,1,1
```